tests/blob_tx_test.go

```diff
@@ -176,6 +178,15 @@ func createTestBlobTransactionWithNilSidecar(t *testing.T, ctxt *testContext) (*
 	return types.SignTx(tx, types.NewCancunSigner(chainId), ctxt.net.validator.PrivateKey)
 }
 
+func lastBlockSanity(t *testing.T, client *ethclient.Client) {
```

Comment:
nit: should we check all blocks up to the last, just to make sure no new blocks are inserted between the one created with the blob transaction and the one that is tested here?

Fetching those blocks should take no time.

Reply:
done